`cargo update` to sgx 1.1.5 fails CI

As topic, steps to reproduce:
- git clone this repo and `cd sgx-runtime`
- `cargo update` => it will update the SGX dep to 1.1.5
- `cargo check -p test-no-std`

It fails with the errors:
```
error[E0658]: use of unstable library feature 'available_parallelism'
  --> /home/kai/.cargo/git/checkouts/incubator-teaclave-sgx-sdk-f208006cdf70a903/605c8ea/sgx_unwind/build.rs:92:13
   |
92 |             thread::available_parallelism()
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #74479 <https://github.com/rust-lang/rust/issues/74479> for more information
   = help: add `#![feature(available_parallelism)]` to the crate attributes to enable
```

root cause seems to be sgx_unwind 1.1.5 compilation.

Is it known? (or upstream issue?)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,13 +7,6 @@ members = [
     'substrate-sgx/externalities',
     'test-no-std'
 ]
-[patch."https://github.com/apache/teaclave-sgx-sdk.git"]
-sgx_tstd = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master"}
-sgx_types = { version = "1.1.4", git = "https://github.com/haerdib/incubator-teaclave-sgx-sdk", branch = "master"}
-
-#[patch."https://github.com/integritee-network/pallets.git"]
-#pallet-parentchain = { path = '../pallets/parentchain' }
-
 
 [profile.release]
 panic = 'unwind'

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2021-11-10"
+channel = "nightly-2022-03-10"
 targets = ["wasm32-unknown-unknown"]
 profile = "default" # include rustfmt, clippy

